### PR TITLE
[CI][unittests] Convert json test output to xml

### PR DESCRIPTION
**Pull request by rundong08**
Fixes #<issue_number_goes_here>

> It's a good idea to open an issue first for discussion.

- [ ] Tests pass
- [ ] Appropriate changes to README are included in PR

### CODE REVIEW

File: kokoro/unittests/dart_test_json2xml.py [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+
+  def compute_timings(self, time_zero):
+    ms_since_start = self.pop_info('time')
+    assert ms_since_start is not None
+    start_time = time_zero + timedelta(milliseconds=ms_since_start)
+    self._attrs['timestamp'] = start_time.strftime(self._ISO8601_FORMAT)
+    self._attrs['time'] = '{:.3f}'.format(
+        (self.end_time - ms_since_start) / 1000)
+
+  def finalize_attrs(self) -> None:
```
Comment (BobEvans) [inline kokoro/unittests/dart_test_json2xml.py]: Finalize seems to really be to_string. Something like that would be a more descriptive name.

**Review by BobEvans — CHANGES_REQUESTED** [review]
Looks good. One comment on a method name and needs tests but otherwise, code looks ready to me.

File: kokoro/unittests/dart_test_json2xml.py (hunk 1 above)
Comment (rundong08) [reply kokoro/unittests/dart_test_json2xml.py]: Thanks! I have changed it into `convert_attrs_to_str`

**Review by BobEvans — APPROVED** [review]
Thanks Rundong! Looks great.

**Pull request merged**